### PR TITLE
Update 2022-Sep-12 with news from Neovim breaking changes and Reddit

### PR DESCRIPTION
**Pull request by echasnovski**
There were updates in 'awesome-neovim', but all of new entries seem to be far older than "this week".

### CODE REVIEW

File: contents/2022/Sep/12.md [hunk 1]
```diff
@@ -236,6 +266,69 @@ A plugin that shows the LSP progress inside [lualine](https://github.com/nvim-lu
...
+  </a>
+</h3>
+
+![compiler-explorer.nvim](https://camo.githubusercontent.com/1dbb63cb80319e0db0fad30114cbb698991856484f24898ff65dbafbd406e418/68747470733a2f2f692e696d6775722e636f6d2f447937546e55642e676966)
+
+[@krady21] released their first Neovim plugin! Now it is possible to interact with
+[compiler-explorer](https://godbolt.org/). Leverages Neovim's `vim.ui`, `vim.notify` and `vim.diagnostic`.
+
+- [Reddit](https://www.reddit.com/r/neovim/comments/x5tajn/compilerexplorernvim_lua_plugin_for_interacting/)
+- [GitHub](https://github.com/krady21/compiler-explorer.nvim)
```
Comment (phaazon) [inline contents/2022/Sep/12.md:286]: This is in the previous week. Please try to ensure the news you add hasn’t been talked about in the previous weeks.

File: contents/2022/Sep/12.md [hunk 2]
```diff
@@ -290,6 +383,27 @@ The plugin now has a merge tool! Also, that screenshot looks really nice, doesn
...
+  </a>
+</h3>
+
+The most famous Lua plugin manager continues getting better. Users can now do `PackerUpdate --preview` to first preview
+possible changes, choose which plugins to actually update and then continue, instead of first updating and then checking
+changes. Very nice contribution by [@AckslD].
+
+- [Reddit](https://www.reddit.com/r/neovim/comments/x5uqfv/preview_flag_added_to_packerupdate/)
+- [GitHub](https://github.com/wbthomason/packer.nvim)
+- [PR](https://github.com/wbthomason/packer.nvim/pull/975)
```
Comment (phaazon) [inline contents/2022/Sep/12.md:403]: Same thing, it was in the weekly last week.

**Review by phaazon — COMMENTED** [review]
You added a bench of stuff that was mentioned last week. Please do a little bit of research before opening contributions like this.

**Comment by echasnovski** [comment]
Strange, I usually do this. Must have missed it this time.

But on the same note, it does start to get hard keeping up with what was added previously and what wasn't. For updates, it is simpler (just checking previous week will cover most of the cases). But for new plugins it does require a full search every time, because there are different sources where plugin is announced as "new" and they might not be synchronized. Maybe it is more of a question about what plugin can qualify as "new". I am not sure this is easily fixable, just thinking out loud.

**Pull request closed without merging**